README.md

```diff
@@ -101,15 +101,15 @@ The following devices are supported by this version of liquidctl.  See each guid
 |:--- |:---:|:--- |
 | Gigabyte RGB Fusion 2.0 Motherboards | [guide](docs/gigabyte-rgb-fusion2-guide.md) | <sup>_E_</sup> |
 | EVGA GTX 1080 FTW (only lighting) | [guide](docs/nvidia-guide.md) | <sup>_EUXN_</sup> |
+| ASUS Strix RTX 2080 Ti OC | [guide](docs/asus-rog-guide.md) | <sup>_EUXN_</SUP> |
 | NZXT HUE 2, HUE 2 Ambient | [guide](docs/nzxt-hue2-guide.md) | |
 
-<sup>_L_</sup> _Requires the `--legacy-690lc` flag._  
-<sup>_Z_</sup> _Requires replacing the device driver [on Windows](#installing-on-windows)._  
-<sup>_E_</sup> _Experimental and/or partial support._  
-<sup>_U_</sup> _Requires `--unsafe` features._  
-<sup>_X_</sup> _Only supported on Linux._  
-<sup>_N_</sup> _New driver, only available on git._  
```

Comment:
The two trailing spaces are significant in Markdown, and prevented these notes from being processed as paragraph.

Reply:
oops, my text editor tried to be helpful and removed all trailing spaces. Back to vim to put them back.